pkg/core/bootstrap/bootstrap.go

```diff
@@ -257,7 +258,14 @@ func initializeResourceStore(cfg kuma_cp.Config, builder *core_runtime.Builder)
 		pluginName = core_plugins.Memory
 		pluginConfig = nil
 	case store.PostgresStore:
-		pluginName = core_plugins.Postgres
+		switch cfg.Store.Postgres.DriverName {
+		case postgres.DriverNamePgx:
+			pluginName = core_plugins.Pgx
+		case postgres.DriverNamePq:
+			pluginName = core_plugins.Postgres
+		default:
+			return errors.Errorf("unknown driver name %s", cfg.Store.Postgres.DriverName)
+		}
```

Comment:
What is the reason for keeping pgx as a separate plugin? Could we just return a proper ResourceStore implementation from the Postgres plugin?